### PR TITLE
folder option to imports

### PR DESCRIPTION
**Pull request by memsharded**
To improve: https://github.com/conan-io/conan/issues/1316
Implements also: https://github.com/conan-io/conan/issues/1334

### CODE REVIEW

File: conans/client/loader_parse.py [hunk 1]
```diff
@@ -110,32 +110,56 @@ def options(self):
...
+
+            invalid_line_msg = "Invalid imports line: %s\nEX: OpenCV/lib, * -> ./lib" % line
+            if line.startswith("/") or line.startswith(".."):
+                raise ConanException("%s\n%s" % (invalid_line_msg,
+                                                 "Import's paths can't begin with '/' or '..'"))
             try:
-                if local_install_line.startswith("/") or local_install_line.startswith(".."):
-                    raise ConanException("Import's paths can't begin with '/' or '..'")
-                pair = local_install_line.split("->")
+                tokens = line.split("@", 1)
```
Comment (lasote) [inline conans/client/loader_parse.py]: Maybe an auxiliary function to get the parameters would be more readable.

**Comment by lasote** [comment]
Very cool!

File: conans/client/loader_parse.py (hunk 1 above)
Comment (memsharded) [inline conans/client/loader_parse.py]: Done

**Comment by lasote** [comment]
Oh, it's broken. Fix it and merge.

**Comment by lasote** [comment]
All gut ;)

**Pull request merged**